tenable/reports/nessusv2.py

```diff
@@ -54,6 +54,8 @@ def _defs(self, name, value):
 
         elif name in ['cvss_base_score', 'cvss_temporal_score']:
             # CVSS scores are floats, so lets return them as such.
+            if value is None:  # Issue #551
+                return 0
             return float(value)
```

Comment:
```suggestion
            if value:
                return float(value)
```

Reply:
> Sorry for the delay. I was waiting for the engineering team that took this over to comment and take the appropriate action.

No worries, I appreciate you taking the time to review. Also, I agree with that proposed behavior